{
  try {
    Assert.assertEquals(StorageResultCode.OperationSucceeded,createContainer("testRenameContainer.1",4,"none"));
    Assert.assertEquals(StorageResultCode.OperationSucceeded,createContainer("testRenameContainer.2",4,"none"));
    Assert.assertEquals(StorageResultCode.OperationFailedStorageMounted,renameContainer("testRenameContainer.1","testRenameContainer.2"));
  }
 catch (  Exception e) {
    failStr(e);
  }
}
